portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -16,20 +16,45 @@
 
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.Arrays;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   static final long serialVersionUID = 1L;
 
+  /**
+   * Constant test comments
+   */
+  static final String[] TEST_COMMENTS = {
```

Comment:
nit: no reason to create a separate variable than the `commentsList`. Could write `commentsList = Arrays.asList("This is a normal comment", "...", "...")`.

Reply:
Change accepted.